src/ResourceManager/Billing/Commands.Billing.Test/ScenarioTests/EnrollmentAccountTests.ps1

```diff
@@ -0,0 +1,39 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+List billing periods
+#>
+function Test-ListEnrollmentAccounts
+{
+    $enrollmentAccounts = Get-AzureRmEnrollmentAccount
```

Comment:
The setup that is necessary to run the test should be part of the test.

Reply:
Unfortunately there's no simple way to create enrollment accounts via PowerShell. These currently have to be created via the EA portal, and this is something a department admin would need to do. Additionally, the owner of the enrollment account will need to accept the invitation to complete the workflow.

I'll go ahead and document the steps required in the test, but I don't believe it's possible for us to automate this at this time.

Reply:
Thanks, the comments will help